### PR TITLE
Krig Goodies & Bighorn Edits

### PR DESCRIPTION
**Pull request by ProbablyCarl**
- - -
Balance:
 - Followers given a proper botanical lab, sealed from the outside world. For everyone's safety.
 - Fermi chems and a few others can no longer be generated through botany.
 - CQC is no longer hit with 'no guns', and instead gets poor aim, identically to Berserker Rights.
 - 30 MINUTE lock added to the Casper Mountain Complex area. For both sides.
 - Legion provided M2 Browning within a crate. This crate is located in their armory.
 - Previous Super Mutant Overlord location redone to have a bunch of melee weapons, for the Tribals.

Alternatively, in regards to followers, it was summed up by a player pretty well:
"...botany lab, overpass to bring people past the botany lab, morgue was removed and replaced with a graveyard+undertaker cabin, workshop got QoL and space to expand, storage area added to workshop, Tesla Armor (Fallout 1) was added to workshop..."

- - -
Map:
 - Followers given a botanical lab.
 - Town provided a small growing plot just beneath the old area they'd grow with the Followers.
 - Brah-Min returned. Hopefully it snows later.
 - Brahmin herd moved to Brah-Min's domain.
- - -

### CODE REVIEW

**Comment by ProbablyCarl** [comment]
image

**Comment by ProbablyCarl** [comment]
Going to speed merge my own PR, since I can't very well can't anyone else to review it. Mostly.
Lets hope I didn't screw up. Probably good to go. _Probably._

File: code/__DEFINES/melee.dm [hunk 1]
```diff
@@ -11,3 +11,4 @@
 #define MARTIALART_BERSERKER "berserker"
 #define MARTIALART_RISINGBASS "rising bass"
 #define MARTIALART_RANGERTAKEDOWN "ranger takedown"
+#define MARTIALART_KRIGSERKER "ranger takedown"
```
Comment (PolymorphBlue) [inline code/__DEFINES/melee.dm:14]: I think this needs to be fixed

File: code/__DEFINES/reagents.dm [hunk 2]
```diff
@@ -80,3 +80,21 @@
 //Chemical reaction flags, for determining reaction specialties
 #define REACTION_CLEAR_IMPURE       (1<<0)  //Convert into impure/pure on reaction completion
 #define REACTION_CLEAR_INVERSE      (1<<1)  //Convert into inverse on reaction completion when purity is low enough
+
+//Pulled from POOL blacklist. Instead of the pool being closed due to stingraids, we'll close the plant section of the market due to Fermichem.
+GLOBAL_LIST_INIT(blacklisted_plant_reagents, list(
+	/datum/reagent/toxin/plasma, /datum/reagent/oxygen, /datum/reagent/nitrous_oxide, /datum/reagent/nitrogen,		//gases
```
Comment (PolymorphBlue) [inline code/__DEFINES/reagents.dm:86]: Newline issue, it looks like?  A little bit more of 'why' would be nice too

File: code/modules/hydroponics/plant_genes.dm [hunk 3]
```diff
@@ -156,6 +157,8 @@
 /datum/plant_gene/reagent/can_add(obj/item/seeds/S)
 	if(!..())
 		return FALSE
+	if(reagent_id in GLOB.blacklisted_plant_reagents)
```
Comment (PolymorphBlue) [inline code/modules/hydroponics/plant_genes.dm:160]: Do we want to provide any response as to why it's returning false?

File: modular_citadel/code/modules/client/loadout/__donator.dm [hunk 4]
```diff
@@ -183,10 +183,19 @@
...
 /obj/item/storage/box/large/custom_kit/shark/PopulateContents()
 	new /obj/item/book/granter/crafting_recipe/happysharky(src)
 	new /obj/item/reagent_containers/food/snacks/sunset/happyshark/jerky(src)
 	new /obj/item/reagent_containers/food/snacks/sunset/happyshark/candybar(src)
 	new /obj/item/reagent_containers/food/drinks/sunset/happyshark/bittercola(src)
+
+/datum/gear/donator/kits/krig
+	name ="Krig's Teachings"
+	path = /obj/item/storage/box/large/custom_kit/krig
+	ckeywhitelist = list ("lia526")
```
Comment (PolymorphBlue) [inline modular_citadel/code/modules/client/loadout/__donator.dm:197]: Should we really have ckeys embedded in code? Particularly as a donator item?

File: code/modules/reagents/chemistry/reagents/toxin_reagents.dm [hunk 5]
```diff
@@ -51,7 +51,7 @@
 	overdose_threshold = 18 // So, someone drinking 20 units will FOR SURE get overdosed
 	taste_description = "horrific agony"
 	taste_mult = 0.9
-	var/datum/disease/fev_disease = /datum/disease/transformation/mutant
+	var/datum/disease/fev_disease = null
```
Comment (PolymorphBlue) [inline code/modules/reagents/chemistry/reagents/toxin_reagents.dm:54]: Why are these being disabled?

**Pull request merged**